src/Dropbox.elm

```diff
@@ -950,6 +951,8 @@ program config =
                     Just m ->
                         config.update m model
                             |> Update.Extra.mapCmd (Msg << Just)
-        , subscriptions = \_ -> Sub.none
+        , subscriptions = \model ->
+            config.subscriptions model
+                |> Platform.Sub.map (Msg << Just)
```

Comment:
Sorry for the nitpicky comment, but along with the import change above, this can become
```
subscriptions = config.subscriptions >> Sub.map (Msg << Just)
```

Reply:
Done.